models/test_models.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+import numpy as np
+from brainscore.benchmarks._neural_common import average_repetition, timebins_from_assembly
+from brainscore.metrics.ceiling import InternalConsistency
+from brainscore.metrics.transformations import Split
+
+from brainscore.metrics.regression import CrossRegressedCorrelation, pls_regression, pearsonr_correlation
+
+from brainio_base.assemblies import NeuroidAssembly
+from brainio_base.stimuli import StimulusSet
+from brainscore.benchmarks import BenchmarkBase, ceil_score
+from brainscore.metrics import Score
+from brainscore.model_interface import BrainModel
+from model_tools.brain_transformation import ModelCommitment
+
+
+def test_brain_models(module):
+    module = __import__(module)
+    for model in module.get_model_list():
+        layers = module.get_layers(model)
+        assert layers is not None
+        assert isinstance(layers, list)
+        assert len(layers) > 0
+        model = module.get_model(model)
+        assert model is not None
+        assert isinstance(model, BrainModel)
+        test_brain_model_processing(model, module)
+    print('Test successful, you are ready to submit!')
+
+
+def test_brain_model_processing(model, module):
+    # to be done
+    return
+
+
+def test_base_models(module):
+    module = __import__(module)
+    for model in module.get_model_list():
+        layers = module.get_layers(model)
+        assert layers is not None
+        assert isinstance(layers, list)
+        assert len(layers) > 0
+        assert module.get_model(model) is not None
+        test_processing(model, module)
+        print('Test successful, you are ready to submit!')
+
+
+def test_processing(model, module):
+    os.environ['RESULTCACHING_DISABLE'] = '1'
+    model_instance = module.get_model(model)
+    layers = module.get_layers(model)
+    brain_model = ModelCommitment(identifier=model, activations_model=model_instance,
+                                  layers=layers, region_benchmarks={'IT': _MockBenchmark()})
```

Comment:
these 4 `test_*` methods are detected by `pytest` as unit tests and fail since there is no `module` fixture.
They exist to ensure the model submission works right? It might be better to move this file into `brainscore.submission` and replace `test_` with `check_`